includes/Core/API/ApiRouter.php

```
@@ -0,0 +1,884 @@
+				'data'    => array(),
+			)
+		);
+	}
+
+	/**
+	 * Update preset.
+	 *
+	 * @param \WP_REST_Request $request The REST API request.
+	 * @return \WP_REST_Response The response object.
+	 */
+	public function update_preset( \WP_REST_Request $request ): \WP_REST_Response {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)